examples/nextjs/starter/src/app/layout.tsx

```diff
@@ -11,14 +12,13 @@ export default function Layout( { children }: PropsWithChildren ) {
 }
 
 /**
- * Generate custom metadata with rootMetadata generated using generateRootMeraData.
- *
- * @return dynamic metadata generated from generateRootMetaData() and custom logic.
+ * Generate site meta data.
+ * @return Metadata for SEO.
  */
 export async function generateMetadata(): Promise< Metadata > {
-	const rootMetaData = await generateRootMetaData();
+	const metaData = await getSiteMetadata();
 
 	return {
-		...rootMetaData,
+		...metaData,
 	};
```

Comment:
```suggestion
	const metadata = await getSiteMetadata();

	return {
		...metadata,
	};
```

Reply:
Huh?

Reply:
Its a camelCase nit. The IRL name is `metadata` not `meta[space]data` and this brings the variable in line with `getSiteMetadata` (lowercase `d`)

Reply:
> (lowercase d)

Could not see that in the review before. Got it.